exercise_0/terminalMode/TerminalMode.h

```diff
@@ -0,0 +1,22 @@
+//
+// Created by emyrmak on 7/3/2019.
+//
+
+#ifndef CHARACTERCOUNTERAPP_TERMINALMODE_H
+#define CHARACTERCOUNTERAPP_TERMINALMODE_H
+#include "../characterCounter/CharacterCounter.h"
```

Comment:
Please avoid relative includes. You should set an include path properly. You can achieve that by adding such code to CMake (after creating the `exercise_0` target):
```cmake
target_include_path( exercise_0 ${CMAKE_CURRENT_SOURCE_DIR})
```

And then you could just `#include "characterCounter/CharacterCounter.h"`